LTHPasscodeViewController/LTHPasscodeViewController.m

```diff
@@ -1060,6 +1063,61 @@ - (void)showLockScreenWithAnimation:(BOOL)animated withLogout:(BOOL)hasLogout an
     }
 }
 
+- (void)showLockScreenIntoSuperview:(UIView *)superview WithAnimation:(BOOL)animated withLogout:(BOOL)hasLogout andLogoutTitle:(NSString*)logoutTitle {
```

Comment:
This is basically the same code as `showLockScreenWithAnimation` just that it uses the `view` passed in instead of `LTHMainWindow`, right? Can we extract this code into a method, so it's not duplicated? For example `showLockScreenWithAnimation` could call this method by passing `LTHMainWindow` as a parameter.